examples/vg-specs/text_scatter_colored.vg.json

```diff
@@ -88,7 +88,7 @@
                                 "field": "Origin"
                             },
                             "align": {
-                                "value": "center"
+                                "value": "left"
```

Comment:
I am not sure... But i have rebased and it still have the same output as this. It could be my mistake that have this spec from the other branch and pushed it here...